src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,28 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+
+import { Todo } from '../../types/Todo';
+import users from '../../api/users';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={classNames('TodoInfo', {
+        'TodoInfo--completed': todo.completed,
+      })}
+    >
+      <h2 className="TodoInfo__title">
+        {todo.title}
+      </h2>
+
+      {users.map((user) => (user.id === todo.userId && (
+        <UserInfo user={user} key={user.id} />
+      )))}
```

Comment:
it's not a good approach to find user like this. better use `find` method. here you can just:
```ts
const todoOwner: User | undefined = users.find((user) => user.id === todo.userId)
```

Reply:
and then:
```ts
{!!todoOwner && <UserInfo user={todoOwner} key={todoOwner.id} />}
```

Reply:
I didn't test it localy, but it should work